### DISCUSSION TITLE
It's some error happend when i was trying to archive my app

### DISCUSSION THREAD

**Discussion by yongbeomkwak** [krzyzanowskim/CryptoSwift#1031] [Q&A] [answered]
Hello guys 

When I run the app in xcode, there is no error, but when I archive it, an error appears.

i use the Tuist for build a project

please let me know if you know  a solution
<img width="379" alt="스크린샷 2023-11-23 오후 7 01 00" src="https://github.com/krzyzanowskim/CryptoSwift/assets/48616183/51f9e778-af9c-4763-b48f-ec7a6ebf1fa5">

**Comment by krzyzanowskim** [comment, marked as answer]
that only means something is off with the xcode setup. not much to be done here.
